kolibri/core/assets/src/state/actions.js

```diff
@@ -642,8 +642,13 @@ function saveAttemptLog(store) {
 }
 
 function saveAndStoreAttemptLog(store) {
+  const attemptLogId = store.state.core.logging.attempt.id;
+  const attemptLogItem = store.state.core.logging.attempt.item;
+  const storeAttemptLog = () =>
```

Comment:
maybe call this `isSameAttempt`?

some more comments in here would be helpful - there's a lot of useful info in the PR description.

is the idea here that unsaved attempts don't have IDs?

Reply:
Yeah, I can copy some stuff over.

Unsaved attempts don't have ids, no.